SdkVersion.props

```diff
@@ -7,11 +7,12 @@
     <SDKVersionRS4>10.0.17134.0</SDKVersionRS4>
     <SDKVersionRS5>10.0.17763.0</SDKVersionRS5>
     <SDKVersion19H1>10.0.18362.0</SDKVersion19H1>
-    <SDKVersionInsider>10.0.21681.0</SDKVersionInsider>
+    <SDKVersion21H1>10.0.22000.0</SDKVersion21H1>
+    <SDKVersionInsider>10.0.22000.0</SDKVersionInsider> <!-- Matches SDKVersion21H1 -->
   </PropertyGroup>
   <PropertyGroup>
-    <!-- By default we use the publicly shipped SDK version which is 19H1 now -->
-    <MuxSdkVersion Condition="$(UseInsiderSDK) != 'true'">$(SDKVersion19H1)</MuxSdkVersion>
+    <!-- By default we use the publicly shipped SDK version which is 21H1 now -->
+    <MuxSdkVersion>$(SDKVersion21H1)</MuxSdkVersion>
```

Comment:
I would keep the "Condition="$(UseInsiderSDK) != 'true'"" part so that when we need a new insider SDK in the future, we'll just need to set SDKVersionInsider to some higher number, and it will just work.